lib/CL/CMakeLists.txt

```diff
@@ -196,6 +196,7 @@ set(POCL_LIB_SOURCES  "clCreateContextFromType.c"
                    "clEnqueueSVMMemcpyRectPOCL.c"
                    "clEnqueueSVMMemfillRectPOCL.c"
                    "clSetKernelArgDevicePointer.c"
+                   "clCreateBuiltinKernelWithAttributes.cc"
```

Comment:
Let's keep the runtime implementations in C for minimal implicit overheads for low end edge devices and soft cores.